src/modules/sector-industries/entities/sector-industry.entity.ts

```diff
@@ -87,15 +87,15 @@ export class SectorIndustryEntity {
   @ApiProperty({ example: '2010-03-15 00:00:00.000' })
   created: Date;
 
-  @Column({ name: 'DATE_LAST_UPDATED_DATETIME' })
-  @ApiProperty({ example: 0, description: '2010-03-15 00:00:00.000' })
+  @Column({ name: 'DATE_LAST_UPDATED_DATETIME', type: 'timestamp' })
+  @ApiProperty({ example: '2010-03-15 00:00:00.000' })
```

Comment:
Result is in format: 2017-04-01T00:00:00.000Z
Example could be in same format.

Reply:
good shout - updated